appveyor.yml

```diff
@@ -4,7 +4,7 @@ install:
       Add-Type -AssemblyName System.IO.Compression.FileSystem
       if (!(Test-Path -Path "C:\sbt" )) {
         (new-object System.Net.WebClient).DownloadFile(
-          'https://piccolo.link/sbt-0.13.18.zip',
+          'https://piccolo.link/sbt-1.3.8.zip',
```

Comment:
should this be 1.4.7 to match the build.properties?

Reply:
agree, probably switch to https://github.com/sbt/sbt/releases/download/v1.4.7/sbt-1.4.7.zip following the example https://github.com/sbt/sbt/issues/5860#issuecomment-698335998 since piccolo returns 404 for 1.4.x

Reply:
switched